pkgs/by-name/ne/neohtop/package.nix

```diff
@@ -0,0 +1,68 @@
+{
+  rustPlatform,
+  cargo-tauri,
+  nodejs,
+  jq,
+  stdenv,
+  fetchNpmDeps,
+  zsh,
+  fetchFromGitHub,
+  npmHooks,
+}:
+
+let
+  tauri = cargo-tauri;
+
+in
+rustPlatform.buildRustPackage rec {
+  name = "neohtop";
+  pname = name;
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "abdenasser";
+    repo = "neohtop";
+    rev = "v${version}";
+    hash = "sha256-23r/oBR4zWmeSVa5QaJ2wJOwzwmqYVhMKOpaEABP7Qs=";
+  };
+  nativeBuildInputs = [
+    tauri.hook
+    npmConfigHook
+    nodejs
+  ];
+  cargoRoot = "src-tauri";
+  cargoLock.lockFile = ./Cargo.lock;
+  npmDeps = fetchNpmDeps {
+    inherit src;
+    hash = "sha256-6FWowWVVeU7U5aM4WGVWv0WmauLv/F4mtSdjNeeF6eQ=";
+  };
+  cargoHash = "";
```

Comment:
Why?

Reply:
because this does not work (cargo cannot find reqwest) I think its related to:
https://github.com/HannesGitH/nixpkgs/blob/c43b2b1ea412767dd01e4354f4ca05ff942df4e4/pkgs/by-name/su/surrealist/package.nix#L52-L57